packages/monorepo-releaser/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "monorepo-releaser",
+  "version": "0.0.0",
+  "bugs": "https://github.com/IBM/test-automation/issues",
+  "description": "Tools and configs for releasing monorepo packages via semantic-release",
+  "main": "index.js",
+  "author": "IBM Watson Health Pattern & Asset Library (https://github.com/orgs/IBM/teams/whpal/members)",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:IBM/test-automation.git"
+  },
+  "license": "Apache-2.0",
+  "engines": {
+    "node": "^12"
+  },
+  "scripts": {
+    "semantic-release": "semantic-release --debug",
+    "test": "true"
+  },
+  "release": {
+    "extends": [
+      "semantic-release-monorepo",
+      "./release.config"
+    ]
+  },
+  "dependencies": {
+    "semantic-release": "17.0.8",
```

Comment:
semantic-release's documentation says it should be a devDependency for a package. Is there a reason you have it under dependencies?  I don't see it imported anywhere.

Reply:
this package is intended to be the source for all release functionality for all other packages. This is legal, but outside of the default intended usage for this module. These three modules are required to be installed when `monorepo-releaser` is a dependency, so they must be in `dependencies` because `devDeps` are not installed when a module is installed as a `dep`.

Reply:
I guess they sound more like peer dependencies than dependences to me. Those packages aren't technically part of what's distributed with these configurations.

Reply:
these dependencies are included from a technical standpoint as full-dependencies very much on purpose. This will allow any node module which has `monorepo-releases` as a `dependency` _or_ a `devDep` to not then have to have (or track updates from) semantic-release or semantic-release-monorepo as dependenices. If those two modules are `peerDependencies` in `monorepo-releases`, then that requires ingesting modules to _also_ have both modules as dependencies and track changes and updates to those modules. The point of this module is to have essentially zero configuration or maintenance when it comes to creating releases.

This is the same concept as [`react-scripts`](https://blog.logrocket.com/everything-you-need-to-know-about-react-scripts/) which is automatically adds to `npm scripts` when using `create-react-app`. The users of that module only need to track updates to react-scripts - but not all the underlying modules. Test, build, start, etc, all are created and maintained by `react-scripts`

```
 "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  }
```

The point of this is that the end-user wants to build and maintain their react application without having to develop and maintain an entire development environment. 

`monorepo-releases'` point is to remove the need to develop and maintain release functionality, dependency updates and configurations. When `semantic-release` updates to 18x, what if the configuration options change? In this instance, an ingesting module might _never_ know that semrel had an update or that 18x was a breaking change to the config. The module just updates `monorepo-releases`, which will give the ingesting app _both_ the latest semrel and semrel-monorepos versions _and_ the config-change which accounts for the breaking change - all by only changing the version of `monorepo-releases`.